packages/mask/src/components/shared/ApplicationSettingPluginSwitch.tsx

```diff
@@ -80,10 +80,10 @@ export const ApplicationSettingPluginSwitch = memo(({ focusPluginId }: Props) =>
     const targetPluginRef = useRef<HTMLLIElement | null>()
     const noAvailablePlugins = availablePlugins.length === 0
 
-    useLayoutEffect(() => {
+    useEffect(() => {
         if (!focusPluginId || noAvailablePlugins || !targetPluginRef.current) return
         targetPluginRef.current.scrollIntoView()
-    }, [focusPluginId, noAvailablePlugins])
+    }, [focusPluginId, noAvailablePlugins, targetPluginRef])
```

Comment:
No need to have a ref in dependency array because it doesn't notify React when it value got changed.